src/components/layout/Layout.scss

```diff
@@ -1,26 +1,48 @@
 @import '../../styles/variables';
+@import '../../styles/mixins';
 
 .layout {
-    min-width: 456px;
-    max-width: 456px;
-    margin: 10% auto;
+    width: 456px;
+    margin: auto;
     padding: 24px 22px;
     border-radius: 6px;
     background-color: $white;
     box-shadow: 0 1px 24px 0 rgba($black, 0.04);
 
+    &__container {
+        height: calc(100% - #{$header});
```

Comment:
something's wrong
![image](https://user-images.githubusercontent.com/1554577/56568943-b23d5180-65c0-11e9-8400-6cce201eb2a4.png)

Reply:
i think this would work just fine, and will remove the need for calc() 
![image](https://user-images.githubusercontent.com/1554577/56569177-2d9f0300-65c1-11e9-8861-008dcdbc0190.png)

Reply:
Thanks, fixed